package.json

```diff
@@ -340,7 +340,7 @@
     "@cypress/skip-test": "^2.6.1",
     "@cypress/webpack-preprocessor": "^5.9.1",
     "cypress": "8.7.0",
-    "puppeteer": "^2.1.1",
```

Comment:
There's really no need to change the version of puppeteer we use during our CI runs, this does not impact users.

FYI we're migrating away from puppeteer to playright, so getting this updated and dealing with breaking changes in puppeteer is really just not worth any time at all.